### PR TITLE
Fix code-style instead of turning off rules

### PR DESCRIPTION
**Pull request by wooorm**

### CODE REVIEW

File: examples/react-static/src/pages/404.js [hunk 1]
```diff
@@ -1,7 +1,9 @@
 import React from 'react'
 
-export default () => (
+const notFound = () => (
```
Comment (ChristianMurphy) [inline examples/react-static/src/pages/404.js:3]: initial cap ins conventional for component names

```suggestion
const NotFound = () => (
```

File: packages/preact/test/test.jsx [hunk 2]
```diff
@@ -125,11 +125,13 @@ test('should support `wrapper` in `components`', async () => {
 test('should support dots in component names (such as `ol.li`) for a direct child “selector”', async () => {
   const Content = await run('* a\n1. b')
 
+  const olLi = props => <li className="ordered" {...props} />
```
Comment (ChristianMurphy) [inline packages/preact/test/test.jsx:128]:
```suggestion
  const OlLi = props => <li className="ordered" {...props} />
```

File: examples/react-static/src/pages/404.js [hunk 3]
```diff
@@ -1,7 +1,9 @@
 import React from 'react'
 
-export default () => (
+const NotFound = () => (
   <div>
     <h1>404</h1>
   </div>
 )
+
+export default notFound
```
Comment (wooorm) [inline examples/react-static/src/pages/404.js:9]:
```suggestion
export default NotFound
```

File: packages/preact/test/test.jsx [hunk 4]
```diff
@@ -125,11 +125,13 @@ test('should support `wrapper` in `components`', async () => {
...
   const Content = await run('* a\n1. b')
 
+  const OlLi = props => <li className="ordered" {...props} />
+
   assert.equal(
     render(
       <MDXProvider
         components={{
-          'ol.li': props => <li className="ordered" {...props} />
+          'ol.li': olLi
```
Comment (wooorm) [inline packages/preact/test/test.jsx:134]:
```suggestion
          'ol.li': OlLi
```

File: packages/react/test/test.jsx [hunk 5]
```diff
@@ -136,11 +136,13 @@ test('should support `wrapper` in `components`', async () => {
 test('should support dots in component names (such as `ol.li`) for a direct child “selector”', async () => {
   const Content = await run('* a\n1. b')
 
+  const olLi = props => <li className="ordered" {...props} />
```
Comment (wooorm) [inline packages/react/test/test.jsx:139]:
```suggestion
  const OlLi = props => <li className="ordered" {...props} />
```

File: packages/react/test/test.jsx [hunk 6]
```diff
@@ -136,11 +136,13 @@ test('should support `wrapper` in `components`', async () => {
...
+
   assert.equal(
     renderToString(
       <MDXProvider
         components={{
-          'ol.li': props => <li className="ordered" {...props} />
+          'ol.li': olLi
```
Comment (wooorm) [inline packages/react/test/test.jsx:145]:
```suggestion
          'ol.li': OlLi
```

**Review by ChristianMurphy — APPROVED** [review]

**Pull request merged**